### PR TITLE
feature/1_2_add_orion_connection_and_append

### PR DESCRIPTION
**Pull request by frbattid**
- Implements issues #1 and #2

### CODE REVIEW

File: doc/ngsi/orion/user_and_programmer_guide/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#User and Programmer Guide
+
+* [Introduction](./introduction.md)
+* [Creating an Orion connection](./creating_orion_connection.md)
+* [Appending data to Orion](./appending_data.md)
+* [Querying data from Orion](./querying_data.md)
+* [Issues and contact information](./issues_and_conact.md)
```
Comment (pcoello25) [inline doc/ngsi/orion/user_and_programmer_guide/README.md]: conact -> contact

File: doc/ngsi/orion/user_and_programmer_guide/creating_orion_connection.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Start by creating a new job if you did not create it yet:
+
+![Create a job](./images/create_job.png)
+
+In the palette, within the package <i>NGSI/Orion</i>, there is a Talend component, `tOrionConnection`, in charge of defining all the required details for an Orion connection:
+
+* Log4j log file.
+* Log4j level.
+* Http-like Orion service endopoint, typically running on TCP/1026 port.
```
Comment (pcoello25) [inline doc/ngsi/orion/user_and_programmer_guide/creating_orion_connection.md]: endopoint sounds funny :laughing:

File: doc/ngsi/orion/user_and_programmer_guide/README.md (hunk 1 above)
Comment (frbattid) [inline doc/ngsi/orion/user_and_programmer_guide/README.md]: Fixed in PR https://github.com/telefonicaid/iotp-talend-connectors/pull/4/commits/91793586c59de32feacb932ee31347e94d71284d

File: doc/ngsi/orion/user_and_programmer_guide/creating_orion_connection.md (hunk 2 above)
Comment (frbattid) [inline doc/ngsi/orion/user_and_programmer_guide/creating_orion_connection.md]: Fixed in PR https://github.com/telefonicaid/iotp-talend-connectors/pull/4/commits/91793586c59de32feacb932ee31347e94d71284d

**Comment by pcoello25** [comment]
LGTM

**Pull request merged**